test/Mvc6Framework45.FunctionalTests/project.json

```diff
@@ -56,12 +56,8 @@
   ],
   "scripts": {
     "postrestore": [
-      "npm install",
-      "node_modules/.bin/flatten-packages", 
-      "node_modules/.bin/bower install"
```

Comment:
DNX is complaining about the postrestore and prepare scripts. It says that NPM is not recognized as an internal command!